web/src/js/components/FlowView/Messages.jsx

```diff
@@ -68,17 +68,18 @@ function ResponseLine({ flow, readonly, updateFlow }) {
 
 const Message = connect(
     state => ({
-        flow: state.ui.flow.modifiedFlow || state.flows.byId[state.flows.selected[0]],
+        flow: state.flows.byId[state.flows.selected[0]],
```

Comment:
`flow: state.ui.flow.modifiedFlow || state.flows.byId[state.flows.selected[0]],` changes the flow prop, and then `<ContentLoader ..> <View.... > </ContentLoader> will be rerendered and the content will be reloaded.